manifests/overlays/kubeflow-platform/kustomization.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+sortOptions:
+  order: fifo
+resources:
+  - ../../third-party/jobset # Comment this line if JobSet is installed on the Kubernetes cluster.
+  - ../../base/crds
+  - ../../base/manager
+  - ../../base/rbac
+  - ../../base/runtimes/pretraining
```

Comment:
Shall we install runtimes right after the `../../base/crds` since we don't need to wait for `manager` to start ?